src/parse.d

```diff
@@ -231,6 +231,20 @@ struct PrefixAttributes
     const(char)* comment;
 }
 
+/*****************************
+ * Destructively extract storage class from pAttrs.
+ */
+private StorageClass getStorageClass(PrefixAttributes* pAttrs)
```

Comment:
I agree. I would have named it removeStorageClass or  popStorageClass (like pop from queue).